scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -337,9 +337,11 @@ muts <- c(match.df$AA_mut)
 # get bulk frequency values
 bulk <- as.numeric(match.df$freq)
 
-ifelse (is.null(dim(muts)),
-  executeDeconvolution <- as.logical("FALSE"), 
-  executeDeconvolution <- as.logical("TRUE" ))
+if (length(muts)==0){
+  executeDeconvolution <- as.logical("FALSE")
+} else{
+  executeDeconvolution <- as.logical("TRUE" )
+}
```

Comment:
see my earlier comment on how to express this more simply.

Generally, it's better to only assign once and compute the value in a conditional instead of assigning in two different places.